### PR TITLE
Optimize: file /cpuset slice make cap

### PR DESCRIPTION
**Pull request by demoManito**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
I think slice should initialize cap, which is a good style
reference code specification: https://github.com/uber-go/guide/blob/master/style.md#specifying-slice-capacity
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: pkg/kubelet/cm/cpuset/cpuset.go [hunk 1]
```diff
@@ -223,7 +226,10 @@ func (s CPUSet) ToSliceInt64() []int64 {
 // ToSliceNoSortInt64 returns a slice of int64 that contains all elements from
 // this set.
 func (s CPUSet) ToSliceNoSortInt64() []int64 {
-	var result []int64
+	if s.IsEmpty() {
+		return nil
+	}
```
Comment (yxxhero) [inline pkg/kubelet/cm/cpuset/cpuset.go:231]: Is this judgment necessary? When the for iteration is empty, can it not be returned correctly?

Comment (demoManito) [reply pkg/kubelet/cm/cpuset/cpuset.go]: This is to keep the logic consistent with the previous one. Uninitialized slice can be returned

**Comment by pacoxu** [comment]
/label tide/merge-method-squash
Would you squash commits?

**Comment by demoManito** [comment]
> /label tide/merge-method-squash Would you squash commits?

commits squashed

**Comment by haircommander** [comment]
/lgtm

nice optimization!

**Review by mrunalp — APPROVED** [review]

**Pull request merged**